### PR TITLE
feat: add Overview first tab for aggregated token details (OK-57944)

### PR DESCRIPTION
**Pull request by franco-chan**
[OK-57944](https://onekeyhq.atlassian.net/browse/OK-57944)

----------

## What

Aggregated (multi-chain) tokens now open the Token Details page on a new **Overview** first tab, modeled on the operations team's request. Single-chain (non-aggregated) tokens are unchanged and have no Overview tab.

The Overview tab shows:
- **Total balance hero** — cross-chain fiat total + token amount, equal to what Home shows for that token.
- **Send / Receive / Swap / Buy** actions with per-action network resolution (Send/Buy → network selector, Receive → aggregate receive selector, Swap → largest-balance member chain).
- **Earn banner** with the whole-symbol max APY; tapping opens the protocol list unfiltered by network.
- **Asset allocation** list — one row per chain (network avatar, name, % of total, balance + fiat, trailing chevron). Tapping a row jumps to that chain's tab and auto-enables/creates the network address if needed. Zero-balance chains stay visible, sunk to the bottom.

## Why

Operations asked for a multi-chain aggregate token to land on a summary view first (Bitget-style), instead of dropping straight into the first chain's history. Users get the total and per-chain split at a glance, with actions that resolve the network in one extra step.

## Notes for reviewers

- **No backend / no new i18n.** `tokenMap` already carries the aggregate total + member fiat at first paint; existing `AggregateTokenSelector` screens cover network selection; Earn APIs already accept an optional `filterNetworkId`. All copy reuses existing `ETranslations` keys.
- **Tab memory removed.** The per-token last-active-tab persistence (`lastActiveTabName` in `ServiceToken` + simpleDb) is deleted so the page always defaults to Overview. `clearLastActiveTabNameData` is kept and still wipes stale persisted data on bootstrap.
- **Shared extractions** (used by both the header and chain tabs): `TokenDetailsBalanceHero`, `pushSwapFromTokenDetails`, `openFiatCryptoWidget`, and the `useAggregateTokenDetails` sort/totals hook.
- Product decisions, edge cases, and QA test cases are tracked in **OK-57944**.
- Structural changes worth a closer look: `index.tsx` `aggregateTabs` descriptor array (replaces scattered tab-offset arithmetic) and `TokenDetailsHeader.pushSwapFromTokenDetails`.

## Testing

`yarn agent:check --profile commit` green (lint + staged typecheck). Runtime self-test on a wallet with multi-chain aggregated balances is pending; QA scenarios are in OK-57944.

### Known v1 scope
- Buy entry is buy-only (sell stays on chain tabs).
- Network dropdown has no "Overview" entry.
- No dedicated overview-exposure / allocation-click analytics events yet.

[OK-57944]: https://onekeyhq.atlassian.net/browse/OK-57944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,546 @@
...
+          !row.tokenDetail &&
+          row.token.accountId &&
+          row.token.networkId &&
+          !backfilledKeysRef.current.has(row.token.$key),
+      );
+      if (!missingRows.length) {
+        return;
+      }
+      missingRows.forEach((row) =>
+        backfilledKeysRef.current.add(row.token.$key),
```
Comment (sidmorizon) [inline packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx:149]: # 🟡 P2 Overview 回填失败后永久不再重试

## 问题

回填逻辑在 `await` 之前就把成员的 `$key` 写入 `backfilledKeysRef`（第 125-127 行），而 `catch` 分支（第 146-148 行）只吞掉异常、不移除该 key。因此只要 `fetchTokensDetails` 发生一次瞬时失败（或返回空 `resp?.[0]`），该成员的 key 就永久留在集合里。后续 `rows` 变化触发 `usePromiseResult` 重跑时，`missingRows` 过滤器（第 120 行 `!backfilledKeysRef.current.has(...)`）会把它排除，`updateTokenDetails` 再也不会被调用。

## 影响

如文件顶部注释所述，成员 tab 只在 focus 时拉取，而用户停留在 Overview 时永不 focus，因此没有任何恢复路径：网络恢复后该行仍停留在 `--` 占位符，Hero 的 `totalFiatValue`/`totalBalanceParsed` 也会永久漏算该成员，直到组件卸载重建。此时该行还会显示 `network_enable_or_create_address` 的刷新图标 tooltip，对一个其实已有地址（`accountId` 存在）、只是拉取失败的成员而言是误导。瞬时网络抖动即可触发。

## 建议

只在成功写入后才标记 key，或在 `catch`（以及 `data` 为空）时把 key 从集合移除，使下次重跑能重试：

```ts
} catch {
  backfilledKeysRef.current.delete(token.$key);
}
```

[Claude session ↗](https://claude.ai/code/session_016Kzhd1MNNAHXcnhjQUn7dH)

---
_Generated by [Claude Code](https://claude.ai/code)_

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/index.tsx [hunk 2]
```diff
@@ -556,31 +542,89 @@ function TokenDetailsView() {
...
+      (aggregateTabs
+        ? aggregateTabs[activeTabIndex]?.token
+        : tokens[activeTabIndex]) ?? tokens[0];
+    if (!activeToken) return;
+
+    const resp = await backgroundApiProxy.serviceToken.fetchTokenInfoOnly({
+      networkId: activeToken.networkId ?? '',
+      tokenAddress: activeToken.address,
+    });
+    updateTokenMetadata({
```
Comment (originalix) [inline packages/kit/src/views/AssetDetails/pages/TokenDetails/index.tsx]: 快速切换 Overview 和成员链 Tab 时，较早的 `fetchTokenInfoOnly` 请求可能最后返回。这里在请求体内直接调用 `updateTokenMetadata`，会绕过 `usePromiseResult` 对过期 result 的 nonce 保护，导致 footer 又显示前一个成员链的市场数据。

另外这里没有写入 `resp.currency`。Overview 首屏没有成员 Header 补充该字段，非默认法币下 `Currency` 会把接口价格当成当前展示币种，金额会被错误解释。

请让 `usePromiseResult` 返回完整元数据，并只用最新 result 更新 context，同时包含 `currency: resp?.currency`。

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsDeFiBlock.tsx [hunk 3]
```diff
@@ -71,6 +77,10 @@ export function TokenDetailsDeFiBlock({
   const isUnmountedRef = useRef(false);
 
   const cacheKey = `${networkId}_${tokenAddress}`;
```
Comment (originalix) [inline packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsDeFiBlock.tsx]: 聚合模式下，查询结果取决于 `aggregateTokens` 的成员网络集合；但当前 `cacheKey` 只有聚合 mock `networkId` 和 `$key`。同一 main runtime 内切换到网络能力不同的钱包时，会复用前一个钱包缓存的协议、最高 APY，甚至缓存的 `null`，导致 Earn 入口错误显示或消失。

请在聚合模式下把排序后的成员 `networkId/address` 指纹加入 `cacheKey`，让不同成员集合拥有独立缓存项。

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx [hunk 4]
```diff
@@ -0,0 +1,546 @@
...
+      screen: EAssetSelectorRoutes.AggregateTokenSelector,
+      params: {
+        accountId,
+        indexedAccountId,
+        aggregateToken: tokenInfo,
+        aggregateSubTokenList: tokens,
+        enableNetworkAfterSelect: true,
+        closeAfterSelect: false,
+        onSelect: async (token: IAccountToken) => {
+          await openFiatCryptoWidget({
```
Comment (originalix) [inline packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx:419]: 当用户在聚合 Buy 选择器中选到不支持法币购买的成员网络时，这里会直接调用 `openFiatCryptoWidget`。该 helper 不执行 `serviceFiatCrypto.isTokenSupported`，因此仍会请求 `generateWidgetUrl`，用户最终只能遇到失败或通用错误；原有单链 `ActionBuy` 会先做支持性检查。

请在 `onSelect` 中先校验该成员的 `buy` 支持状态，不支持时阻止 widget 请求并给出明确反馈，仅支持时再调用 `openFiatCryptoWidget`。

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx (hunk 1 above)
Comment (franco-chan) [reply packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx:149]: Fixed in 910ac57f1a — the key is now removed from `backfilledKeysRef` on catch and on an empty `resp?.[0]`, so the next `rows`-triggered run retries the member instead of leaving it on the placeholder forever.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/index.tsx (hunk 2 above)
Comment (franco-chan) [reply packages/kit/src/views/AssetDetails/pages/TokenDetails/index.tsx]: Fixed in 910ac57f1a — the hook now returns the metadata (so usePromiseResult's stale-run protection drops out-of-order responses) and a useEffect writes only the latest result into the context, now including `currency: resp?.currency`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsDeFiBlock.tsx (hunk 3 above)
Comment (franco-chan) [reply packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsDeFiBlock.tsx]: Fixed in 910ac57f1a — aggregate mode now appends a sorted `networkId:address` member fingerprint to `cacheKey`, so different member sets (e.g. wallets with different network capabilities) get independent cache entries.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

File: packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx (hunk 4 above)
Comment (franco-chan) [reply packages/kit/src/views/AssetDetails/pages/TokenDetails/TokenDetailsOverview.tsx:419]: Fixed in 910ac57f1a — `onSelect` now awaits `serviceFiatCrypto.isTokenSupported({type: 'buy'})` for the picked member first; unsupported networks show a "{networkName} network is not supported yet." toast and skip the widget request.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

**Review by originalix — APPROVED** [review]
